assembly/NFT/NFT_fortest.ts

```diff
@@ -0,0 +1,237 @@
+///////////////////////////////
```

Comment:
Sorry but i think it is pointless to run tests on a modified copy of your code like this... If things are missing then we have to add it. But for me this workaround is not acceptable